adafruit_emc2101/__init__.py

```diff
@@ -410,7 +410,7 @@ def spinup_time(self, spin_time):
         # Not importing at top level so the SpinupTime is not loaded
         # unless it is required, and thus 1KB bytecode can be avoided.
         # pylint: disable=import-outside-toplevel
-        from emc2101_enums import SpinupTime
+        from .emc2101_enums import SpinupTime
```

Comment:
```suggestion
        from adafruit_emc2101.emc2101_enums import SpinupTime
```
Is this the same or better ? both work, and i'm not a dev ><